tests/php/Jobs/CloneImagesOrVideosTest.php

```diff
@@ -113,32 +116,34 @@ public function testCloneVideoVolume()
     public function testCloneVolumeImages()
     {
         Event::fake();
-        $volume = $this
-            ->volume([
-            'media_type_id' => MediaType::imageId(),
-            'created_at' => '2022-11-09 14:37:00',
-            'updated_at' => '2022-11-09 14:37:00',
-            ])
-            ->fresh(); // Use fresh() to load even the null fields.
+        $volume = 
+            $this
+                ->volume([
+                    'media_type_id' => MediaType::imageId(),
+                    'created_at' => '2022-11-09 14:37:00',
+                    'updated_at' => '2022-11-09 14:37:00',
+                ])
+                ->fresh(); // Use fresh() to load even the null fields.
```

Comment:
No need for a newline after `$this`.

```suggestion
            $this->volume([
                'media_type_id' => MediaType::imageId(),
                'created_at' => '2022-11-09 14:37:00',
                'updated_at' => '2022-11-09 14:37:00',
            ])
            ->fresh(); // Use fresh() to load even the null fields.
```